exporting/opentsdb/metadata.yaml

```diff
@@ -0,0 +1,156 @@
+          detailed_description: |
+            The format of each item in this list, is: [PROTOCOL:]IP[:PORT].
+            - PROTOCOL can be udp or tcp. tcp is the default and only supported by the current exporting engine.
+            - IP can be XX.XX.XX.XX (IPv4), or [XX:XX...XX:XX] (IPv6). For IPv6 you can to enclose the IP in [] to separate it from the port.
+	    - PORT can be a number of a service name. If omitted, the default port for the exporting connector will be used (opentsdb = 4242).
+
+            Example IPv4:
+	      ```yaml
+              destination = 10.11.14.2:4242 10.11.14.3:4242 10.11.14.4:4242
+	      ```
+            Example IPv6 and IPv4 together:
+            ```yaml
+            destination = [ffff:...:0001]:2003 10.11.12.1:2003
+            ```
+            When multiple servers are defined, Netdata will try the next one when the previous one fails.
+        - name: 'username'
+          default_value: 'my_username'
+          description: 'Username for HTTP authentication'
+          required: false
+        - name: 'password'
+          default_value: 'my_password'
+          description: 'Password for HTTP authentication'
+          required: false
+        - name: 'data source'
+          default_value: ''
+          description: 'Selects the kind of data that will be sent to the external database. (as collected|average|sum)'
+          required: false
+        - name: 'hostname'
+          default_value: '[global].hostname'
+          description: 'The hostname to be used for sending data to the external database server.'
+          required: false
+        - name: 'prefix'
+          default_value: 'Netdata'
+          description: 'The prefix to add to all metrics.'
+          required: false
+        - name: 'update every'
+          default_value: '10'
+          description: |
+            The number of seconds between sending data to the external database. Netdata will add some randomness to this number, 
+            to prevent stressing the external server when many Netdata servers send data to the same database. This randomness does
+            not affect the quality of the data, only the time they are sent.
+          required: false
+        - name: 'buffer on failures'
+          default_value: '10'
+          description: |
+            The number of iterations (each iteration is update every seconds) to buffer data, when the external database server is not
+            available. If the server fails to receive the data after that many failures, data loss on the connector instance is expected
+            (Netdata will also log it).
+          required: false
+        - name: 'timeout ms'
+          default_value: '2 * update_every * 1000'
+          description: 'The timeout in milliseconds to wait for the external database server to process the data.'
+          required: false
+        - name: 'send hosts matching'
+          default_value: 'localhost *'
+          description: |
```

Comment:
I am concerned about the length of the descriptions that may make the table look ugly. But I guess we don't have much choice. :)

Reply:
An alternative is a simple description (that answers the question "What is it?" - e.g. "Hosts filter. The syntax is [simple patterns](https://github.com/netdata/netdata/tree/master/libnetdata/simple_pattern#simple-patterns).") and details in detailed_description. I think you tend to add details to the non-detailed description to "description".

Reply:
An example of using this option should be added to the "Examples config" list:

```yaml
- name: Using `send hosts matching`
  description: ...
  config: ...
- name: Using `send charts matching`
  description: ...
  config: ...

Reply:
Updated